mpqp/core/circuit.py

```diff
@@ -1134,12 +1134,46 @@ def to_other_language(
                 self.nb_qubits,
             )
         elif language == Language.CIRQ:
-            qasm2_code = self.to_other_language(Language.QASM2)
-            if TYPE_CHECKING:
-                assert isinstance(qasm2_code, str)
-            from mpqp.qasm.qasm_to_cirq import qasm2_to_cirq_Circuit
+            from cirq.circuits.circuit import Circuit as CirqCircuit
+            from cirq.ops.named_qubit import NamedQubit
+            from cirq.ops.identity import I
+
+            cirq_qubits = [NamedQubit(f"q_{i}") for i in range(self.nb_qubits)]
+            cirq_circuit = CirqCircuit()
+
+            for qubit in cirq_qubits:
```

Comment:
I would do this only add the end of the instantiation of the circuit, so we do it only for the qubits that were not used (like in braket). To avoid any useless identity.

Reply:
we need all the qubits for concatenation operation, so we must keep it like this